.github/workflows/advanced.yml

```
@@ -14,7 +14,7 @@ jobs:
           echo "dolor sit amet" > lorem_2.md
           mkdir output  # create output dir
           # this will also include README.md
-          echo "::set-output name=files::$(printf '"%s" ' *.md)"
+          echo "files=$(printf '"%s" ' *.md)" > $GITHUB_OUTPUT
```

I did a bit of grepping around other projects and it seems like no quoting at all is needed here. Given than forcing these quotes is likely to give us some other problems (i.e. the shell globbing is going to expand `*` to a fully escaped shell syntax, I'd like to see us try something that doesn't add another layer that may actually be harmful.

Can I bother you with another round of testing using this:

```suggestion
          echo "files=$(echo *.md)" > $GITHUB_OUTPUT
```

Ideally we would have a test that included files with a space in the name such as `my file.md` not just easy ones like `file1.md`.

---

It seems like the quotes are necessary from my testing.

Using the code:
```bash
echo "files=$(echo *.md)" > $GITHUB_OUTPUT
```
the action works as expected when using filenames without spaces, but produced the following error when there are spaces in filenames:
```
pandoc: file: openBinaryFile: does not exist (No such file or directory)
```
I had tested the previous code with filenames that include spaces and it works as expected

---

Alright fair enough. That's not what I would have concluded [from other usages out there](https://grep.app/search?q=files%3D.%2AGITHUB_OUTPUT&regexp=true) if you've tested it in this context we'll go with that.

Thank for taking the time to test and respond to questions, and thanks for the contribution.